### PR TITLE
Nooriyah lohani

### PR DESCRIPTION
**Pull request by pweschmidt**
the writeup for the episode

### CODE REVIEW

**Comment by vsoch** [comment]
Thank you Peter! I just realized this week is thanksgiving, people in the US get Thursday through Friday off. Maybe we should skip a week?

**Comment by pweschmidt** [comment]
That's a bit short notice. How would this work? That we shift everything by one week for the remainder of this year? I.e. Nooriyah goes live next week (3 Dec) and you the week after (10 Dec) and me again on 17 Dec? Would that sound reasonable?

**Comment by vsoch** [comment]
That would work! And we definitely can still release it, I just don’t think as many US listeners will be online to see the posts, and will mostly be spending time with family. But I could be wrong - I’ll leave it up to you - and yes sorry for not thinking of this earlier! A lot going on right now :)

**Comment by pweschmidt** [comment]
Thanks for staying up so late. The US will certainly be in holiday mode I'd expect. We can shift the episodes by one week as suggested. I will update the Google calendar accordingly, if that's ok?
It would look like this: 3 Dec Nooriyah (Peter); 10 Dec Robert (Vanessa); 17 Dec Maelle (Peter), 24 Dec Tania. And I will start again on 14 Jan with my interview with Olivia Guest. Would that work?

**Comment by vsoch** [comment]
Definitely! Thank you @pweschmidt 👍

File: _posts/2020-11-26-nooriyah-lohani.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Equal Footing"
+author: "@pweschmidt"
+rse: "Nooriyah Lohani"
+phenotype: 
+excerpt: "Let's put RSEs on equal footing with academics and professionals and get them recognised and formal roles, says Nooriyah Lohani from NESI in New Zealand. That requires to reach decision makers. And for that you need to build and foster a strong community first. "
+date: 2020-11-26 8:30:00
```
Comment (vsoch) [inline _posts/2020-11-26-nooriyah-lohani.md]:
```suggestion
date: 2020-12-03 8:30:00
```

File: _posts/2020-11-26-nooriyah-lohani.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    url: https://www.nesi.org.nz
+  - name: Mystery Genes
+    url: https://twitter.com/mysterygenes
+  - name: NeSI twitter
+    url: https://twitter.com/nesi_nz 
+--- 
+In some countries research software engineers have established roles and there are (some) career paths available to them. In New Zealand, however, there is still a way to go. Meet [Nooriyah Lohani](https://twitter.com/mysterygenes), research community advisor at [NESI](https://www.nesi.org.nz). Nooriyah's background is in bio informatics and she recently submitted a paper on forecasting the effect of influenza like viruses on the health system in New Zealand.
+Buddying up with Australia, Nooriyah and colleagues of hers in New Zealand are building a community of RSEs in their countries. Their first big milestone was an Australia/new Zealand RSE conference - entirely virtual in Covid times - with ca 150 attendees. 
+Nooriyah wishes to see RSEs on equal footing with academics and other professionals. Ultimately, the success of achieving this goal hinges on working with decision makers. As Nooriyah says: "we need to reach the decision makers. But to do that we need a strong community backing."
```
Comment (vsoch) [inline _posts/2020-11-26-nooriyah-lohani.md]:
```suggestion
Nooriyah wishes to see RSEs on equal footing with academics and other professionals. Ultimately, the success of achieving this goal hinges on working with decision makers. As Nooriyah says, "We need to reach the decision makers. But to do that we need a strong community backing."
```

File: _posts/2020-11-26-nooriyah-lohani.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+explicit: "no"
+resources:
+  - name: New Zealand eScience Infrastructure
+    url: https://www.nesi.org.nz
+  - name: Mystery Genes
+    url: https://twitter.com/mysterygenes
+  - name: NeSI twitter
+    url: https://twitter.com/nesi_nz 
+--- 
+In some countries research software engineers have established roles and there are (some) career paths available to them. In New Zealand, however, there is still a way to go. Meet [Nooriyah Lohani](https://twitter.com/mysterygenes), research community advisor at [NESI](https://www.nesi.org.nz). Nooriyah's background is in bio informatics and she recently submitted a paper on forecasting the effect of influenza like viruses on the health system in New Zealand.
```
Comment (vsoch) [inline _posts/2020-11-26-nooriyah-lohani.md]:
```suggestion
In some countries research software engineers have established roles and there are (some) career paths available to them. In New Zealand, however, there is still a way to go. Meet [Nooriyah Lohani](https://twitter.com/mysterygenes), research community advisor at [NESI](https://www.nesi.org.nz). Nooriyah's background is in bioinformatics and she recently submitted a paper on forecasting the effect of influenza like viruses on the health system in New Zealand.
```

**Review by vsoch — APPROVED** [review]

**Pull request merged**